packages/superset-ui-chart/package.json

```diff
@@ -35,6 +35,7 @@
     "react": "^15 || ^16"
   },
   "peerDependencies": {
+    "@superset-ui/connection": "^0.7.0",
```

Comment:
This is actually a precursor to the embeddable demo PR but there is no harm adding this now. 🙂

Reply:
why?

Reply:
For local development, `yarn install` or `npm install` don't install `peerDependencies` so adding them to `devDependencies` so it is installed automatically.